packages/components/package.json

```diff
@@ -19,14 +19,19 @@
     "directory": "packages/components"
   },
   "scripts": {
-    "build": "tsup src/index.ts --format esm,cjs --dts --external react",
+    "build": "tsup src/index.ts --format esm,cjs --dts --external react --minify",
     "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
     "dev": "tsup src/index.ts --format esm,cjs --watch --dts --external react",
     "lint": "TIMING=1 eslint src/**/*.ts* --fix",
     "prettier-check": "prettier --check package.json tsconfig.json \"./src/**/*.{tsx,ts,json}\"",
     "test": "jest",
     "tsc": "tsc"
   },
+  "dependencies": {
+    "@localyze-pluto/theme": "*",
+    "@stitches/react": "^1.2.8",
+    "lodash": "^4.17.21"
+  },
```

Comment:
I tried to run the storybook, and I'm always having this error: 

```
ModuleNotFoundError: Module not found: Error: Can't resolve '@localyze-pluto/theme' in '/Users/anavale/src/pluto/packages/components/src/utils'
    at /Users/anavale/src/pluto/node_modules/webpack/lib/Compilation.js:2016:28
    at /Users/anavale/src/pluto/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/Users/anavale/src/pluto/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
  ```
  
Am I missing running some commands?